Watchman.Discord/Areas/UselessFeatures/Controllers/GoogleLmgtfyController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Threading.Tasks;
+using System.Web;
+using Devscord.DiscordFramework.Commons.Exceptions;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Discord.Areas.UselessFeatures.BotCommands;
+
+namespace Watchman.Discord.Areas.UselessFeatures.Controllers
+{
+    public class GoogleLmgtfyController : IController
+    {
+        private readonly MessagesServiceFactory _messagesServiceFactory;
+        private const string LINK_TO_LMGTFY = "https://pl.lmgtfy.com/?q=";
+
+        public GoogleLmgtfyController(MessagesServiceFactory messagesServiceFactory)
+        {
+            this._messagesServiceFactory = messagesServiceFactory;
+        }
+
+        public Task PrintGoogleLmgtfy(GoogleCommand googleCommand, Contexts contexts)
+        {
+            if (string.IsNullOrWhiteSpace(googleCommand.Search))
+            {
+                throw new NotEnoughArgumentsException();
+            }
+            var searchLink = LINK_TO_LMGTFY + HttpUtility.UrlEncode(googleCommand.Search);
```

Comment:
a jakbyś spróbował bez użycia searchLink? w sensie to co jest po = dać od razu w linii 30. Zobacz czy będzie lepiej wyglądać.

Reply:
uważam że tak jest jednak czytelniej
30 linia byłaby dużo dłuższa